core/tests/puppeteer-acceptance-tests/spec/exploration-editor-tests/exploration-publisher-drafter.spec.js

```diff
@@ -0,0 +1,72 @@
+// Copyright 2024 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Acceptance Test for Exploration Creator and Exploration Manager
+ */
+
+const userFactory = require(
+  '../../puppeteer-testing-utilities/user-factory.js');
+const testConstants = require(
+  '../../puppeteer-testing-utilities/test-constants.js');
+
+const DEFAULT_SPEC_TIMEOUT = testConstants.DEFAULT_SPEC_TIMEOUT;
+
+describe('Exploration Publisher, Saver and Drafter', function() {
+  let explorationCreator = null;
+  beforeAll(async function() {
+    explorationCreator = await userFactory.createNewExplorationCreator(
+      'explorationAdm', 'exploration_creator@example.com');
+    explorationVisitor = await userFactory.createNewExplorationCreator(
+      'explorationLearner', 'exploration_visitor@example.com');
+  }, DEFAULT_SPEC_TIMEOUT);
+
+  it('should perform exploration management actions',
+    async function() {
+      await explorationCreator.goToDashboardUrl();
+      await explorationCreator.goToEditorSection();
```

Comment:
Probably switchToEditorTab(), if you're in the exploration editor page.

Reply:
its used to reach editor section after click on 
![image](https://github.com/oppia/oppia/assets/136263179/5ee0fdf6-4575-4d19-9227-fe8d0a999299)
we reach editor tab or editor section, if editor tab looks clear then i will update it.

Reply:
updated ✅,